src/dependency/Pyegi/PythonScripts/[sample] prefix/main.py

```diff
@@ -28,4 +28,4 @@ def sub(line: Line, l: Line):  # ln: line number
     sub(line, line.copy())
 
 
-Pyegi.CreateOutputFile(original="C", placement="O")
+Pyegi.create_output_file(transform_original=Pyegi.Transform.COMMENTED, insert_new=Pyegi.Location.ORIGINAL)
```

Comment:
bellow?

Reply:
there is no Original Location

Reply:
true, but this one computes the location of the original line and puts the generated line(s) below it

Reply:
Where did you define `Pyegi.Location.ORIGINAL` enum?!